husky_extensions/globalize.js

```diff
@@ -57,7 +57,7 @@
                         return key;
                     }
 
-                    return !!translation ? translation : key;
+                    return !!translation || translation === '' ? translation : key;
```

Comment:
Some brackets would probably make more clear what's happening here. And what does it actually do?